WORKSPACE

```diff
@@ -336,3 +336,30 @@ maven_jar(
   name = "com_101tec_zkclient",
   artifact = "com.101tec:zkclient:0.3"
 )
+
+# Google Cloud
+maven_jar(
+  name = "google_api_services_storage",
+  artifact = "com.google.apis:google-api-services-storage:v1-rev108-1.22.0"
+)
+
+maven_jar(
+  name = "google_api_client",
+  artifact = "com.google.api-client:google-api-client:1.22.0"
```

Comment:
Could you create a `google_client_version=1.22.0` at the top of this file and reuse that for each of these so we always upgrade in lock step.